.buildkite/commands/lint.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash -u
+
+echo "--- 🧹 Linting"
+# Run lint on the app module first to ensure that the SARIF file is always generated.
+./gradlew :gravatar-quickeditor:lintRelease
```

Comment:
Because `quickeditor` uses `ui` module, the lint will also check the `ui` module (see: `checkDependencies = true`)

I configured it this way, because I want us to have a single lint report that we can send to GitHub Code Scanning registry. If we uploaded per-module reports, they would override each other.

Reply:
I guess that should work, we just need to keep in mind that this has to be updated if we have new modules.

How would that setup look if we had a new module that depends on the `:ui` but not on `:quickeditor`? 

Could we maybe run the lint against the demo app, but ignore errors/warnings from the demo app module?

Reply:
> Because quickeditor uses ui module, the lint will also check the ui module (see: checkDependencies = true)

You are right. I've rechecked it, and it works as you described. Thanks!

Reply:
> Could we maybe run the lint against the demo app, but ignore errors/warnings from the demo app module?

Neat idea @AdamGrzybkowski , thanks! I think it's doable, with a little tweak in `lint.xml`: [ece18ea](https://github.com/Automattic/Gravatar-SDK-Android/pull/315/commits/ece18eaede2aaab46c95721b10130c41cd9c7e99)

When working on this, I've realised that I had misconfigured `lint.xml` previously. The updated configuration is now correct, but it reports errors. What should we do with them?

Reply:
I missed some uCrop related code in the `:ui` module that was making the Lint to fail (already useful :D). I pushed the changes.